dna/metrics.py

```diff
@@ -15,11 +17,22 @@ def rmse(y_hat, y):
     return mean_squared_error(y, y_hat)**.5
 
 
-def pearson_correlation(y_hat, y):
+def pearson_correlation(y_hat, y, name: str, directory: str):
```

Comment:
Let's keep functions as simple as possible. Making the plots should be separate from the score computation. Also, the score should be included in the plot, as I linked here: https://github.com/byu-dml/d3m-dynamic-neural-architecture/issues/37#issuecomment-501427904.

Reply:
Done

Reply:
Use the "Resolve Conversation" button to let me know that it is resolved.